### DISCUSSION TITLE
Split available seats of one event into two individual parts (Dynamic seats?)

### DISCUSSION THREAD

**Discussion by larsit0** [liedekef/events-made-easy#145] [Q&A]
Hi everyone,
not sure if this is for Q&A or rather belongs into 'Ideas' but here we go:

We're using EME for our sports club and I'm currently looking for a way to split the overall number of available seats into individual (dynamic) groups.
The issue: Let's assume we have 40 seats available for an event. I want to set up a dedicated training session as part of that event. This should be open for signups 3 days before the event starts to make sure everyone who wants to participate has a chance to secure a seat. Let's say 10 people have signed up for that.
The remaining 30 seats for general participation should be open for signups 2 days before the event starts, but whenever someone signs up for dedicated training afterwards, this seat should also be subtracted from the seats available for general participation.

Hope I've made my idea clear and maybe there's already a solution for that, but so far I've had no luck finding one.

Thanks for the ongoing work and support! :)

Kind regards,
Lars

**Comment by liedekef** [comment]
This is a difficult one. Splitting the seats is not needed though in my opinion. I would suggest to use a custom field (eg checkbox) to indicate you want to participate in the trainings-part. And using eme_if (to check how many days are left before the event starts) to include a template (using #_INCLUDE_TEMPLATE) that contains javascript to set that checkbox "checked" (and optionally hide it) for the first day the event "opens" for registration.
